src/RTSolution/CRTM_Active_Sensor.f90

```diff
@@ -374,8 +363,10 @@ SUBROUTINE CRTM_Compute_Reflectivity(Atm         , &  ! Input
     P1 = (M6_MM6 * Wavelength_m**4.0_fp) / (PI**5.0_fp * Kw_2)
     P1 = P1 / dZ_m  ! dZ_m to convert water_content to m/v or cloud water density
     ! Calculate transmittance from top to layer k
+    ! Optical depth is not scaled for zenith angle
     DO k = 1, AtmOptics%n_layers
-       Transmittance(k) = EXP(-TWO * SUM(AtmOptics%optical_depth(1:k)))
+       Transmittance(k) = EXP(-TWO * SUM(AtmOptics%optical_depth(1:k) / &
+                               GeometryInfo%Cosine_Sensor_Zenith))
```

Comment:
did you verify that the optical depth doesn't already contain a zenith angle correction?

Reply:
Yes I varied zenith angle from 0 to 50 but optical_depth didn't change. It seems they just apply the correction when needed.